### PR TITLE
Fix radarr movie types.

### PR DESCRIPTION
**Pull request by davidnewhall**
Adds a few missing fields, fixes `ratings` field in Movie type. Replaces `AddMovieOutput` with `Movie` and adds a proper output to `UpdateMovie()`.

### CODE REVIEW

File: shared.go [hunk 1]
```diff
@@ -99,8 +99,12 @@ type Ratings struct {
 	Votes      int64   `json:"votes"`
 	Value      float64 `json:"value"`
 	Popularity float64 `json:"popularity,omitempty"`
+	Type       string  `json:"type,omitempty"`
 }
 
+// OpenRatings is a ratings type that has a source and type.
+type OpenRatings map[string]Ratings
```
Comment (davidnewhall) [inline shared.go:106]: From the `/api/v3/movie` endpoint came this cool payload... 
```json
"ratings": {
    "imdb": {
      "votes": 16719,
      "value": 5.6,
      "type": "user"
    },
    "tmdb": {
      "votes": 566,
      "value": 7,
      "type": "user"
    },
    "metacritic": {
      "votes": 0,
      "value": 54,
      "type": "user"
    },
    "rottenTomatoes": {
      "votes": 0,
      "value": 69,
      "type": "user"
    }
  }
```

File: radarr/movie.go [hunk 2]
```diff
@@ -118,53 +123,17 @@ type AddMovieOptions struct {
 	SearchForMovie bool `json:"searchForMovie"`
 }
 
-// AddMovieOutput is the data returned when adding a movier.
-type AddMovieOutput struct {
```
Comment (davidnewhall) [inline radarr/movie.go:122]: The only difference between this and `Movie` was the `AddOptions` member. I went ahead and added that to `Movie`.

File: radarr/movie.go [hunk 3]
```diff
@@ -46,12 +46,15 @@ type Movie struct {
...
 	Added                 time.Time           `json:"added,omitempty"`
-	Ratings               *starr.Ratings      `json:"ratings,omitempty"`
+	Ratings               starr.OpenRatings   `json:"ratings,omitempty"`
 	MovieFile             *MovieFile          `json:"movieFile,omitempty"`
 	Collection            *Collection         `json:"collection,omitempty"`
 	HasFile               bool                `json:"hasFile,omitempty"`
 	IsAvailable           bool                `json:"isAvailable,omitempty"`
 	Monitored             bool                `json:"monitored"`
+	Popularity            float64             `json:"popularity"`
+	OriginalLanguage      *starr.Value        `json:"originalLanguage,omitempty"`
```
Comment (davidnewhall) [inline radarr/movie.go:56]:
```
"originalFilePath": "Beast.2022.1080p.AMZN.WEBRip.DD5.1.X.264-EVO/Beast.2022.1080p.AMZN.WEBRip.DD5.1.X.264-EVO.mkv",
"popularity": 2240.269,
 ```

File: radarr/movie.go [hunk 4]
```diff
@@ -91,6 +95,7 @@ type MediaInfo struct {
 	VideoBitrate            int     `json:"videoBitrate"`
 	VideoCodec              string  `json:"videoCodec"`
 	VideoFps                float64 `json:"videoFps"`
+	VideoDynamicRangeType   string  `json:"videoDynamicRangeType"`
```
Comment (davidnewhall) [inline radarr/movie.go:98]: This was a blank string on the movie I looked at.
```
"videoDynamicRangeType": "",
```

File: radarr/movie.go [hunk 5]
```diff
@@ -118,53 +123,17 @@ type AddMovieOptions struct {
...
 type AlternativeTitle struct {
-	MovieID    int          `json:"movieId"`
-	Title      string       `json:"title"`
-	SourceType string       `json:"sourceType"`
-	SourceID   int          `json:"sourceId"`
-	Votes      int          `json:"votes"`
-	VoteCount  int          `json:"voteCount"`
-	Language   *starr.Value `json:"language"`
-	ID         int          `json:"id"`
+	MovieMetadataID int64        `json:"movieMetadataId"`
```
Comment (davidnewhall) [inline radarr/movie.go:128]:
```json
  "alternateTitles": [
    {
      "sourceType": "tmdb",
      "movieMetadataId": 2671,
      "title": "Zvērs",
      "sourceId": 0,
      "votes": 0,
      "voteCount": 0,
      "language": {
        "id": 1,
        "name": "English"
      },
      "id": 18219
    },
    {
      "sourceType": "tmdb",
      "movieMetadataId": 2671,
      "title": "La Bestia",
      "sourceId": 0,
      "votes": 0,
      "voteCount": 0,
      "language": {
        "id": 3,
        "name": "Spanish"
      },
      "id": 18220
    },
    {
      "sourceType": "tmdb",
      "movieMetadataId": 2671,
      "title": "جانور",
      "sourceId": 0,
      "votes": 0,
      "voteCount": 0,
      "language": {
        "id": 1,
        "name": "English"
      },
      "id": 18221
    },
    {
      "sourceType": "tmdb",
      "movieMetadataId": 2671,
      "title": "Žvėris",
      "sourceId": 0,
      "votes": 0,
      "voteCount": 0,
      "language": {
        "id": 24,
        "name": "Lithuanian"
      },
      "id": 18330
    }
```

**Review by Fuochi — APPROVED** [review]

**Pull request merged**